packages/sync/src/Actions.php

```diff
@@ -13,13 +18,36 @@
  * It also binds the action to send data to WPCOM to Jetpack's XMLRPC client object.
  */
 class Actions {
-	static $sender                         = null;
-	static $listener                       = null;
+	/**
+	 * A variable to hold a sync sender object.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @var \Automattic\Jetpack\Sync\Sender
```

Comment:
I usually suggest ditching the leading backslash anywhere, YAGNI.

```suggestion
	 * @var Automattic\Jetpack\Sync\Sender
```